src/actions/genSale/getMintlist.ts

```diff
@@ -0,0 +1,61 @@
+import { fetch } from "cross-fetch";
+import { Maybe, Mintlist, GenSaleConfig } from "../../types";
+
+const extractIPFSHash = (hay: string) => {
+    const regex = /(Qm.+)$/;
+    const matches = regex.exec(hay);
+    if (!matches) {
+        return undefined;
+    }
+    return matches[1];
+};
+
+const defaultIpfsGateway = "https://ipfs.io/ipfs/";
+
+export const getMintlist = async (config: GenSaleConfig) => {
+    let mintlist: Maybe<Mintlist>;
+
+    // fetch via main uri
+    try {
+        const res = await fetch(config.merkleTreeFileUri, {
+            method: "GET",
+        });
+        mintlist = (await res.json()) as Mintlist;
+        return mintlist;
+    } catch (e) {
+        console.error(
```

Comment:
let's use the same logger we use in other places

Reply:
Were not using another logger, in some places we are just throwing errors.

In this case, we shouldnt throw an error because the method will try to get the mintlist from backup sources

Reply:
Same logger we use in other repos* I believe is what @Whytecrowe meant. We use logdown

Reply:
Is this worth it to add? I can only see logdown being used in zNS-sales, and it only adds markdown to the logs. Is that useful for us? Seems unnecessary to me

Reply:
if it is not imported in this repo, then it wouldn't make sense to add it for one line usage.